src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java

```diff
@@ -54,11 +54,13 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(GET, "/event-types/*/partitions/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(GET, "/event-types/*/events/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(GET, "/subscriptions/*/events/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(GET, "/subscriptions/*/cursors/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(PUT, "/subscriptions/*/cursors/**").access(hasScope(eventStreamReadScope))
                     .antMatchers(POST, "/event-types/*/events/**").access(hasScope(eventStreamWriteScope))
                     .antMatchers(DELETE, "/event-types/*/**").access(hasScope(nakadiAdminScope))
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
+                    .antMatchers(GET, "/subscriptions/*/**").access(hasScope(eventStreamReadScope))
```

Comment:
Can we use only ** like this: 
/subscriptions/**
I do not fully understand the difference.